### PR TITLE
policy page

### PR DESCRIPTION
**Pull request by emilyspieler**
https://trello.com/c/9FfgALv8
Desktop and mobile version
Just policy page

### CODE REVIEW

File: app/views/home/policy.html.erb [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+       <div class="row col-md-12 disclosures-bottom">
+         <div class="col-md-12">
+           <hr class="disclosures-bottom__line">
+         </div>
+       </div>
+
+       <div class="col-md-12 disclosures__information">
+         <h1 class="disclosures__one-line">Third-party disclosure</h1>
+           <p>We do not sell, trade, or otherwise transfer to outside parties your Personally Identifiable Information unless we provide users with advance notice. This does not include website hosting partners and other parties who assist us in operating our website, conducting our business, or serving our users, so long as those parties agree to keep this information confidential. We may also release information when it's release is appropriate to comply with the law, enforce our site policies, or protect ours or others' rights, property or safety.
```
Comment (markdamobi) [inline app/views/home/policy.html.erb]: This `<p>` tag is missing a closing tag.

File: app/views/home/policy.html.erb [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+           </p>
+         <h2 class="disclosures__title">According to CalOPPA, we agree to the following:</h2>
+           <p class="disclosures_paragraph-space">Users can visit our site anonymously.</p>
+           <p class="disclosures_paragraph-space">Once this privacy policy is created, we will add a link to it on our home page or as a minimum, on the first significant page after entering our website.</p>
+           <p class="disclosures_paragraph-space">Our Privacy Policy link includes the word &apos;Privacy&apos; and can easily be found on the page specified above.</p>
+           <p class="disclosures_paragraph-space">You will be notified of any Privacy Policy changes:</p>
+             <li> On our Privacy Policy Page </li>
```
Comment (markdamobi) [inline app/views/home/policy.html.erb]: @emilyspieler can you review the indentation of the tags on this page? Indentation usually implies some sort of nesting, but there's a lot of indentation on this page that aren't really nested elements. For instance, the indentation of this `<li>` seems like it should be nested in the `<p>` tag above it, but it isn't. I don't think indentation is needed there. 
I think rubocop might be able to detect some of these things, but we don't have that yet.

Comment (uuunk) [reply app/views/home/policy.html.erb]: @markdamobi yeah rubocop will take care of this. After I merge this in I'll pull master and rerun rubocop before merging my PR to master.

File: app/assets/stylesheets/pages/policy.scss [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+      font-style: normal;
+      font-stretch: normal;
+      line-height: 1.33;
+      letter-spacing: 1px;
+      color: $text-subtitle;
+      margin-left: 1%;
+    }
+    &__information {
+      padding-left: 80px;
+      h1 {
```
Comment (uuunk) [inline app/assets/stylesheets/pages/policy.scss]: @emilyspieler A heads up we'll want to refactor these out to a main style guide that had global styles like these types of titles defined. I think @gcwokeup is working on it.

File: app/views/home/policy.html.erb (hunk 1 above)
Comment (uuunk) [reply app/views/home/policy.html.erb]: good catch!

File: app/views/home/policy.html.erb [hunk 4]
```diff
@@ -0,0 +1,301 @@
+<div class="container policy-page">
```
Comment (sanghun89) [inline app/views/home/policy.html.erb]: no BEM on this one?

**Pull request merged**